.github/workflows/tests.yml

```diff
@@ -85,6 +85,7 @@ jobs:
           - "storage-vm lvm-thin"
           - "storage-vm zfs"
           - storage-volumes-vm
+          - container-copy
```

Comment:
Since you are going to alter this PR at least one more time to address Tom's feedback, would you mind alpha-sorting that list? </nitpick>